.vitepress/config.ts

```diff
@@ -33,60 +33,19 @@ export default defineConfig({
       text: 'Edit this page',
     },
     nav: [
-      { text: 'Guides', link: data.userGuideLinks[0].link },
-      { text: 'Codeflow', link: data.codeflow[0].link },
-      { text: 'API', link: data.apiLinks[0].link },
-      { text: 'WebContainers', link: data.webcontainersLinks[0].link },
-      { text: 'Enterprise', link: data.enterpriseLinks[0].link! },
+      { text: 'Guides', link: defaultGroupLink('userGuide') },
```

Comment:
Always include all link groups in sidebar, and choose which are collapsed based on route.